src/delimiter.js

```diff
@@ -375,9 +375,19 @@ const makeStackedDelim = function(
 
     // To cover the gap create by the overlaps, insert one more repeat element,
     // at a position that juts 0.005 above the bottom of the top element.
-    inners.push({type: "kern", size: shiftOfExtraElement});
-    inners.push(makeInner(repeat, font, mode));
-    inners.push(lap);
+    if ((repeat === "\u239c" || repeat === "\u239f") && repeatCount === 0) {
+        // Parentheses need a short repeat element in order to avoid an overrun.
+        // We'll make a 0.3em tall element from a SVG.
+        inners.push({type: "kern", size: -0.15});
```

Comment:
Why 0.3em? It seems like the existing code uses 0.005em for the overlap shift, so I would have imagined using an 0.01em tall SVG with negative vertical shifts of 0.005em on either side.

Also, it would seem nicer to use the 0.3 (or whatever) constant in `svgdata.leftParenInner[2]` instead of repeating it here.

Reply:
> Why 0.3em?

It's been awhile and I'm not 100% positive I remember this correctly, but I think 0.03em has a pretty big tolerance for adjustment. Since the SVG is used only in this one case where parens barely touch, it could be significantly bigger or smaller and still work.

I opted for the number at the bigger end of the range partly because I think I got the original 0.005em number wrong. I've seen some cases on my own screen where there is still a gap. I did not want to go too small a second time.

Reply:
OK, and sounds safe enough in this case, but then maybe we should adjust the 0.005em number? (Can be in a separate PR effort.)

Do you agree with changing the `0.3` constant with `svgdata.leftParenInner[2]`?

Reply:
> Also, it would seem nicer to use the 0.3 (or whatever) constant in svgdata.leftParenInner[2] instead of repeating it here.

Well, I certainly would do something of that nature if the SVG had multiple use cases. We have some other paths in `svgGeometry.js` that can grow or shrink, and we could have done that. But since this does get used for only one case, I thought that the constant number `0.3em` was easy to read.

I don't feel strongly about this one.

Reply:
Oops, I misunderstood your comment. You were tying this constant to a piece of data from `src/buildCommon.js`, not `svgGeometry.`. Sure. I have no problem with that. I'll make a change.

Reply:
You've got quicker fingers that I do. You have approved this PR. Do you still want me to change the 0.3 constant?

Reply:
Sure, now that we agree, please go ahead. I'll merge after checking the commit.